storage-blob-static-website/entrypoint.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
run `shellcheck` against this script.

![image](https://user-images.githubusercontent.com/25762922/57359552-8eb80080-7195-11e9-8f73-376890c5c59a.png)

There seem to be some webhook issues which didn't trigger the check automatically on your PR, I'll fix that.

Reply:
Good catch. Resolved 👍